components/renku_data_services/search/api.spec.yaml

```
@@ -8,6 +8,45 @@ info:
 servers:
   - url: /api/data
 paths:
+  /search/query:
+    get:
+      summary: Run a search query.
+      parameters:
+        - in: query
+          description: query parameters
+          name: params
+          style: form
+          explode: true
+          schema:
+            $ref: "#/components/schemas/SearchQuery"
+      responses:
+        "200":
+          description: Search results according to the query.
+          headers:
+            x-page:
```

suggestion: Lets just make a note that this may of doing pagination is non-standard but we are keeping it for compatibilty reasons. This is more to prevent other people looking at this code and propagating it further. And there is nothing wrong with how you have done - it is just a matter of style. 

```suggestion
            # NOTE: This is not the standard way we do pagination, but to be compatible witht the current search API we make an exception for the search here
            x-page:
```